packages/koa-shopify-auth/src/auth/client/itp-helper.ts

```diff
@@ -0,0 +1,42 @@
+const itpHelper = `(function() {
```

Comment:
Might be worth putting a comment in that these came from `shopify_app` gem and are tested there